lightning/src/chain/channelmonitor.rs

```diff
@@ -2253,7 +2253,7 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 /// transaction and losing money. This is a risk because previous channel states
 /// are toxic, so it's important that whatever channel state is persisted is
 /// kept up-to-date.
-pub trait Persist<Keys: ChannelKeys>: Send + Sync {
+pub trait Persist<Keys: Sign>: Send + Sync {
```

Comment:
What should we do for cases where `Keys` was used as a type parameter? Maybe best to standardize as `Signer`.

Reply:
Oops, my regexes missed those ones.